client/src/actions/user-actions.js

```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
+
+const config = {
+  headers: {
+    "Content-Type": "application/json",
+  },
+};
+
+export const createInvite = async (userId, emailList, userEmail) => {
+  try {
+    await axios.post(`/user/${userId}/invitation`, { emailList }, config);
+  } catch (err) {
+    console.error(err);
```

Comment:
We probably want to return something here so that we can have some kind of notification of if it was successful or not